### ISSUE TITLE
Unable to remove hooks with no group

### ISSUE THREAD

**Issue by alexherbo2** [mawww/kakoune#3670]
It could be nice to auto-generate a group, so it can be completed with `remove-hooks`.

**Comment by mawww** [comment]
Yeah, I wonder what the best approach would be, hook groups are a bit hacky, maybe we could improve that with a hierarchical structure, similar to how highlighters are specified.
